databricks_template_schema.json

```diff
@@ -1,12 +1,15 @@
 {
   "welcome_message": "Welcome to MLOps Stacks. For detailed information on project generation, see the README at https://github.com/databricks/mlops-stacks/blob/main/README.md.",
-  "min_databricks_cli_version": "v0.209.0",
+  "min_databricks_cli_version": "v0.209.1",
```

Comment:
Would you mind updating the main README to `v0.209.1` as well?

Reply:
done!